CHANGELOG.md

```diff
@@ -5,13 +5,15 @@ Note that since we don't clearly distinguish between a public and private interf
 
 ## [Unreleased]
 
-- MolViewSpec: Support for transparency and custom properties
 - Default to `blended` transparency on iOS due to `wboit` not being supported.
 - Fix direct-volume with fog off (and on with `dpoit`) and transparent background on (#1286)
 - Fix missing pre-multiplied alpha for `blended` & `wboit` with no fog (#1284)
 - Fix backfaces visible using blended transparency on impostors (#1285)
 - Fix StructureElement.Loci.isSubset() only considers common units (#1292)
 - Fix `Scene.opacityAverage` calculation never 1
+- MolViewSpec: Support for transparency and custom properties
+- MolViewSpec: MVP Support for geometrical primitives (mesh, line, label, distance measurement)
```

Comment:
Technically mesh does not have to be geometrical primitive. And line includes distance measurement I think.